python/sqlflow_submitter/db_writer/hive.py

```diff
@@ -86,19 +86,16 @@ def write_hive_table(self):
             hdfs_path = os.getenv("SQLFLOW_HIVE_LOCATION_ROOT_PATH", "/sqlflow")
         else:
             hdfs_path = self.hive_location
-        if self.hdfs_namenode_addr == "":
-            namenode_addr = os.getenv("SQLFLOW_HDFS_NAMENODE_ADDR", "127.0.0.1:8020")
-        else:
-            namenode_addr = self.hdfs_namenode_addr
+        
         # upload CSV to HDFS
         hdfs_envs = os.environ.copy()
         if self.hdfs_user != "":
             hdfs_envs.update({"HADOOP_USER_NAME": self.hdfs_user})
         if self.hdfs_pass != "":
             hdfs_envs.update({"HADOOP_USER_PASSWORD": self.hdfs_pass})
         # if namenode_addr is not set, use local hdfs command's configuration.
```

Comment:
update the comment in this line too?

Reply:
Maybe this comment is suitable for the current implementation.